crates/typst/src/model/par.rs

```diff
@@ -380,13 +380,16 @@ impl Construct for ParLine {
 
 /// Possible line numbering scope options, indicating how often the line number
 /// counter should be reset.
+///
+/// Note that currently the manual reset of the line number counter is not
+/// supported.
```

Comment:
Perhaps a bit clearer:

```suggestion
///
/// Note that, currently, manually resetting the line number counter at
/// arbitrary points of your document is not supported.
```

Reply:
I think the "at arbitrary points of your document" is an unnecessary condition. It seems like "at non-arbitrary points" it _is_ supported (even though I don't know which points would that be). But it's not supported at all. So it can be misleading in this sense.

Reply:
okay